tools/ddsrouter_yaml_validator/ddsrouter_yaml_validator/ddsrouter_config_schema.json

```diff
@@ -12,7 +12,7 @@
                 "version":{
                     "type":"string",
                     "enum":[
-                        "v2.0"
+                        "v3.0"
```

Comment:
The other values should also be accepted right?

Reply:
@rsanchez15 ??
I think we should not, as keeping multiple versions is impossible for a json schema.
I have also encountered some issues using and validating with this tool, and we do not know if it is being used. We may rethink if we want to maintain it :confused: